dalite_xblock/dalite_xblock.py

```diff
@@ -143,6 +153,30 @@ def is_lti_ready(self):
         """Check if this XBlock has all settings so it can connect to the LTI."""
         return all((self.launch_url, self.question_id, self.assignment_id))
 
+    def _render_student_view(self, context, in_studio):
+
+        if in_studio:
+            message = _("No question selected. Please click \"Edit\" and enter the assignment ID and question ID.")
+        else:
+            message = _("This task is not configured yet. You can't finish it right now.")
+
+        fragment = super(DaliteXBlock, self).student_view(context)
+        fragment.add_javascript(loader.load_unicode('public/js/dalite_xblock.js'))
+        fragment.initialize_js('DaliteXBlock')
+
+        if not self.is_lti_ready:
+            fragment.content = u''
+            context.update(self._get_context_for_template())
+            context.update({
+                "message": message
+            })
```

Comment:
Since you've gone to the trouble of parameterising the message, and because getting the LTI passport correct is a bit trickier now, I think it's worth telling the Studio user if there's no valid `launch_url` found.

Could you add an additional check for this, and an additional message?  

@bradenmacdonald Is the suggested "invalid LTI Passport" message below ok? 

e.g.
```python
if not self.is_lti_ready:
    if in_studio:
        if not self.launch_url:
            message = _('Invalid LTI Passport found.  Please go to Advanced Settings and ensure the LTI Passport looks like : "dalite-ng:dalite-xblock:<long hash string>".  Consult your Dalite provider for the correct hash string.')
        else:
            message = _("No question selected. Please click \"Edit\" and enter the assignment ID and question ID.")
    else:
        message = _("This question is not configured yet. Please tell your instructor if you see this error.")

    fragment.content = u''
    context.update(self._get_context_for_template())
    context.update({
        "message": message
    })
....
```